test/test_task_on_kart.py

```diff
@@ -40,15 +43,18 @@ def requires(self):
 @inherits(_DummyTaskB)
 class _DummyTaskC(gokart.TaskOnKart):
     task_namespace = __name__
+
```

Comment:
Hope we can run yapf with Github Actions...

Reply:
Sounds good. I hope you make a PR 👍